frontend/ui/elements/screen_rotation_menu_table.lua

```diff
@@ -60,6 +60,44 @@ When unchecked, the default rotation of the file browser and the default/saved r
             separator = true,
         })
 
+        local function image_rotation_item(setting, label, separator)
+            return {
+                text = label,
+                radio = true,
+                separator = separator or false,
```

Comment:
`separator = separator,` is just fine (should work with `nil` just as with `false`)